examples/memory_test.rs

```diff
@@ -0,0 +1,122 @@
+#![no_std]
+use libtock::memop::*;
```

Comment:
To my best knowledge, it is not recommended to use `*` imports. Those make reviewing the code without an IDE a lot harder as you can no longer easily track which module a function came from. As far as I know, the convention is to import the module, e.g. `uselibtock::memop`, and call the function by qualifying the imported module, i.e. `memop::get_mem_start()`. For me, this makes *reading* the code a lot easier.

Reply:
Ok, I'll fix that.